### DISCUSSION TITLE
21st open maintainers call

### DISCUSSION THREAD

**Discussion by mxinden** [libp2p/rust-libp2p#3976] [General]
Hi everyone,

The rust-libp2p maintainers will meet today (Tuesday) 09:00 UTC. See [libp2p calendar](https://calendar.google.com/calendar/embed?src=libp2p.io_0q9682i3te7eanhe9q7ae1c58g%40group.calendar.google.com).

https://meet.jit.si/rust-libp2p-open-maintainers-call

Everyone is welcome to join, listen along, propose topics and join the discussions.

Please post below in case you already want to add an agenda item.

See you all later!

**Comment by mxinden** [comment]
@arpankapoor in case you want to discuss https://github.com/libp2p/rust-libp2p/pull/3964/.

**Reply by arpankapoor** [reply]
Apologies @mxinden, I'm busy between 09:00-11:00 UTC. I'm open to discuss this over a call at a later time if you'd like.
